src/Palette.ts

```diff
@@ -22,11 +22,12 @@ type PalettePropsType = {
  * Given a palette defined in a json structure, compute the number of rows
  * and columns in that palette.
  *
- * @param {Object} paletteDefinition - An object that lists the positions,
- *                 heights and widths of the cells in the palette.
+ * @param {Object} paletteDefinition - An object in the pre-defined type of
```

Comment:
I have a thought.  Would it be clearer to say `{JsonPaletteType}` instead of `{Object}`?

I admit that I have not been consistent, but I did use `{BciAvIdType}` as one of the `@param`s and `{BlissSVGBuilder}` as the return type in SvgUtlis.ts, see the documentation for the [`getSvgBuilder()`](https://github.com/inclusive-design/adaptive-palette/blob/2132549cc65c4debd79b8efec08165cdefddd333/src/SvgUtils.ts#L47) function.  I did that consistently with SvgUtils.ts, at least.  I did not follow that pattern at all in PaletteStore.ts (unfortunately).

We had as similar approach in GPII where we specified `@param {Component}` to show that the parameter was a fluid component, and a return type of `{Promise}`, which were actually `fluid.promise` objects.

What do you think?  Even if you agree, I don't think we should use this PR to go around and fix all of the documentation.  We go either do it as we edit the files, or make it a separate small task to fix those details.

Reply:
I see your point and fixed the type here.